src/components/TextListItem.js

```diff
@@ -1,19 +1,23 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import { StyleSheet } from 'react-native';
 import { ListItem } from 'react-native-elements';
 
 import { colors, normalize } from '../config';
 import { arrowRight } from '../icons';
 import { Icon } from './Icon';
+import { Image } from './Image';
+
 import { RegularText, BoldText } from './Text';
 import { Touchable } from './Touchable';
 import { trimNewLines } from '../helpers';
 
 export class TextListItem extends React.PureComponent {
   render() {
     const { navigation, noSubtitle, item } = this.props;
-    const { routeName, params, subtitle, title, bottomDivider, topDivider } = item;
+    const { routeName, params, subtitle, title, bottomDivider, topDivider, picture } = item;
 
+    let leftImage = false;
```

Comment:
this `leftImage` needs a comment if it stays or be removed. the information for showing the left image needs to come from props, which will be based on the user preference.